src/common/dao/pgsql.go

```diff
@@ -92,10 +96,13 @@ func (p *pgsql) Register(alias ...string) error {
 		p.host, p.port, p.usr, p.pwd, p.database, p.sslmode)
 
 	if err := orm.RegisterDataBase(an, "pgx", info, orm.MaxIdleConnections(p.maxIdleConns),
-		orm.MaxOpenConnections(p.maxOpenConns), orm.ConnMaxLifetime(5*time.Minute)); err != nil {
+		orm.MaxOpenConnections(p.maxOpenConns), orm.ConnMaxLifetime(p.connMaxLifetime)); err != nil {
 		return err
 	}
 
+	db, _ := orm.GetDB(an)
```

Comment:
We'd better check error here to avoid db is nil then will panic in the next line.

Reply:
@chlins Fixed.